src/NServiceBus.Transport.Msmq/JournalOptionExtensions.cs

```diff
@@ -18,8 +19,18 @@ public static class JournalOptionExtensions
         public static void UseJournalQueue(this ExtendableOptions options, bool enable = true)
         {
             Guard.AgainstNull(nameof(options), options);
-            var ext = options.GetExtensions();
-            ext.Set(KeyJournaling, enable);
+
+            options.GetDispatchProperties()[KeyJournaling] = enable.ToString();
```

Comment:
Nitpicking: do we need to pass invariant culture here, just in case?

Reply:
see previous comment